start-minecraftFinalSetup

```diff
@@ -111,21 +111,21 @@ EOF
     fi
 
     cd "${FTB_DIR}"
-    echo "Running FTB ${FTB_SERVER_START} in ${FTB_DIR} ..."
+    log "Running FTB ${FTB_SERVER_START} in ${FTB_DIR} ..."
     if isTrue ${DEBUG_EXEC}; then
       set -x
     fi
-    exec mc-server-runner ${mcServerRunnerArgs} "${FTB_SERVER_START}"
+    exec mc-server-runner ${mcServerRunnerArgs} "${FTB_SERVER_START}" | sed "s/^/[minecraft] /"
 else
   # If we have a bootstrap.txt file... feed that in to the server stdin
   if [ -f /data/bootstrap.txt ]; then
     bootstrapArgs="--bootstrap /data/bootstrap.txt"
   fi
 
-  echo "Starting the Minecraft server..."
+  log "Starting the Minecraft server..."
   JVM_OPTS="-Xms${INIT_MEMORY} -Xmx${MAX_MEMORY} ${JVM_OPTS}"
   if isTrue ${DEBUG_EXEC}; then
     set -x
   fi
-  exec mc-server-runner ${bootstrapArgs} ${mcServerRunnerArgs} java $JVM_XX_OPTS $JVM_OPTS $expandedDOpts -jar $SERVER "$@" $EXTRA_ARGS
+  exec mc-server-runner ${bootstrapArgs} ${mcServerRunnerArgs} java $JVM_XX_OPTS $JVM_OPTS $expandedDOpts -jar $SERVER "$@" $EXTRA_ARGS | sed "s/^/[minecraft] /"
```

Comment:
Since it's an `exec` I'm not sure if piping to `sed` actually works...or worse, it might default the point of the `exec` and insert a shell process as a parent of `mc-server-runner` process. Can you provide a `ps -ef` of a container running with these changes so I can confirm the process tree?

Reply:
It works for sure. At least my output looks perfectly fine:
```
2020-03-03T16:30:15.739032751Z  [wrapper] Copying any mods over...
2020-03-03T16:30:15.742490670Z  [wrapper] Setting initial memory to 1G and max to 1G
2020-03-03T16:30:15.742728666Z  [wrapper] Starting the Minecraft server...
2020-03-03T16:30:18.490584942Z  [minecraft] [16:30:18] [main/INFO]: Loading for game Minecraft 1.15.2
2020-03-03T16:30:18.563249509Z  [minecraft] [16:30:18] [main/INFO]: [FabricLoader] Loading 2 mods: minecraft@1.15.2, fabricloader@0.7.8+build.184
```

As for how it actually runs:
```
$ docker exec 8234897ee6ee ps -ef
PID   USER     TIME  COMMAND
    1 minecraf  0:00 {start-minecraft} /bin/bash /start-minecraftFinalSetup
   38 minecraf  0:00 mc-server-runner --stop-duration 60s java -XX:+UseG1GC -Xms1G -Xmx1G -jar fabric-server-launch.jar
   39 minecraf  0:00 sed s/^/[minecraft] /
   46 minecraf  1:24 java -XX:+UseG1GC -Xms1G -Xmx1G -jar fabric-server-launch.jar
  106 root      0:00 ps -ef
```

I'll be perfectly honest and say that I don't quite follow what issues might arise from doing it the way I'm doing it. Are you thinking about performance?

Reply:
Sorry, I didn't realize `ps` by default on alpine doesn't show the parent PID of each process. Please re-run with

```
ps -ef -o pid,ppid,args
```

The problem I'm worried about is signal propagation, specifically the SIGTERM when the container is stopped. The main reason I created mc-server-runner was to intercept the SIGTERM and send a `stop` command to gracefully stop the minecraft server since it itself doesn't properly handle SIGTERM. [This article](https://medium.com/@gchudnov/trapping-signals-in-docker-containers-7a57fdda7d86) explains it pretty nicely.

For comparison, here's what the process tree looks like on master:
```
PID   PPID  COMMAND
    1     0 mc-server-runner --stop-duration 60s java -XX:+UseG1GC -Xms1G -Xmx1G -jar minecraft_server.1.15.2.jar
   56     1 java -XX:+UseG1GC -Xms1G -Xmx1G -jar minecraft_server.1.15.2.jar
```

where mc-server-runner must become PID 1 to intercept the signals.

So...long story, short, I think it'll take an enhancement in mc-server-runner to properly prepend those logs. FWIW, I have been wanting to enrich those logs with a date, in addition to time, so that would be a reasonable modification.